python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -133,11 +134,16 @@ def __init__(
         self._dome_az_in_position: typing.Union[None, asyncio.Event] = None
         self._dome_el_in_positio: typing.Union[None, asyncio.Event] = None
 
-        self.m1m3_raise_timeout = 600.0  # timeout to raise m1m3, in seconds.
+        # timeout to raise m1m3, in seconds.
+        self.m1m3_raise_timeout = 600.0
 
         # Tolerance on the stability of the balance force magnitude
         self.m1m3_force_magnitude_stable_tolerance = 50.0
 
+        self._m1m3_actuator_id_index_table: dict[int, int] = dict(
+            [(fa[1], fa[0]) for fa in FATABLE]
```

Comment:
FATABLE provides location of the indices - FATABLE_INDEX,,. to get actuator id from index, you do FATABLE[index][FATABLE_ID]., At least replacing [1] with FATABLE_xxx makes sense, creating that as function will be even better (or shall be done later in ts_cRIOpy)

Reply:
Sorry, not sure I understood your comment. 

I see that I can get the actuator id from the FATABLE index but what I need to do here is the opposite, e.g. get the index for a particular actuator id.

Reply:
Oh! I see! I should use `FATABLE_INDEX` instead of hardcode the indices! nice! thanks